kong/tracing/instrumentation.lua

```diff
@@ -201,12 +199,12 @@ function _M.request(ctx)
     http_flavor = string.format("%.1f", http_flavor)
   end
 
-  local active_span = tracer.start_span(span_name, {
+  local active_span = tracer.start_span("kong.request", {
     span_kind = 2, -- server
     start_time_ns = start_time,
     attributes = {
-      ["http.method"] = method,
-      ["http.url"] = req_uri,
+      ["http.method"] = method, -- part of resource
+      ["http.url"] = req_uri, -- part of resource
```

Comment:
What is this comment supposed to mean?

Reply:
in my opinion we should get rid of these comments, but it's indicating that those attributes will be used by datadog-tracing to generate the _resource_ name